### ISSUE TITLE
react/prop-types warns about destructured spread `other` properties

### ISSUE THREAD

**Issue by rocketraman** [yannickcr/eslint-plugin-react#46]
Sometimes a component does not care about various properties it receives -- they are just passed on to children as-is, and as per the FB docs the canonical way to do this is via a destructuring assignment with a spread operator on the last variable.

`prop-types` warns that props validation is missing for that `other` variable:

```
warning  'other' is missing in props validation  react/prop-types
```

I don't think `react/prop-types` should raise a warning about these "other" properties.
